packaging/observability/microshift-observability.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=MicroShift Observability
+After=microshift.service
+
+[Service]
+ExecStart=/usr/bin/opentelemetry-collector --config=/etc/microshift/opentelemetry-collector.yaml
+Restart=always
+User=root
+TimeoutStartSec=60
+
+# It takes a bit for the certs to be created. This service will reach it's burst limit almost immediately, pretty much
+# guaranteeing that it will reach the restart limit before it can possibly succeed.
+RestartSec=200ms
```

Comment:
Do we really need this? We've configured the service to start `After` microshift, so microshift must report readiness to systemd before the current service startup is attempted. MicroShift only reports readiness after creating all certificates.
What am I missing?